contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -760,4 +744,41 @@ abstract contract PoSValidatorManager is
         PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
         return $._posValidatorInfo[validationID].owner != address(0);
     }
+
+    function _withdrawValidationRewards(address rewardRecipient, bytes32 validationID) internal {
+        PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+
+        uint256 rewards = $._redeemableValidatorRewards[validationID];
+        delete $._redeemableValidatorRewards[validationID];
+
+        _reward(rewardRecipient, rewards);
+    }
+
+    function _withdrawDelegationRewards(
+        address rewardRecipient,
+        bytes32 delegationID,
+        bytes32 validationID
+    ) internal returns (uint256, uint256) {
+        PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+
+        uint256 delegationRewards = 0;
+        uint256 validatorFees = 0;
```

Comment:
OpenZeppelin called out in their recent audit that _not_ initializing variables with their default value is more gas efficient.
```suggestion
        uint256 delegationRewards;
        uint256 validatorFees;
```

Reply:
That's insane that this doesn't compile down to the same bytecode... will fix!